manifests/vanilla/validatingwebhook.yaml

```diff
@@ -116,8 +116,19 @@ spec:
         role: vsphere-csi-webhook
     spec:
       serviceAccountName: vsphere-csi-webhook
-      nodeSelector:
-        node-role.kubernetes.io/control-plane: ""
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
```

Comment:
why do we have `node-role.kubernetes.io/control-plane` and `node-role.kubernetes.io/controlplane`

Did we have k8s releases with such labels?